README.md

```diff
@@ -21,112 +29,166 @@ If you think you've found a bug or would like to suggest an enhancement, please
-  interface.
-  ```
-  192.168.10.254  admin-management admin.ceph salt
-  192.168.10.11   mon1-management mon1.ceph
-  192.168.10.21   data1-management data1.ceph
-
-  172.16.1.254    admin-public admin admin.ceph
-  172.16.1.11     mon1-public mon1 mon1.ceph
-  172.16.1.21     data1-public data1 data1.ceph
-
-  172.16.2.254    admin-cluster admin.ceph
-  172.16.2.11     mon1-cluster mon1.ceph
-  172.16.2.21     data1-cluster data1.ceph
-  ```
-- `/etc/salt/master.d/interface.conf` in our example would be
-  ```
-  # IP address of admin node on management network
-  interface: 192.168.10.254
-  ```
-- After accepting the Salt keys on the master, you can execute the command
-  `salt-key --list-all` on the `admin node`, and you should see that each minion
-  is listed under `Accepted Keys` and that each minion has the same id as
-  is demonstrated for our theoretical cluster. (Each entry under `Accepted
-  Keys` is a `minion id` by which Salt and DeepSea will target the node)
-  ```
-  > salt-key --list-all
-  Accepted Keys:
-  admin.ceph
-  data1.ceph
-  mon1.ceph
-  Denied Keys:
-  Unaccepted Keys:
-  Rejected Keys:
-  ```
-
-### Install DeepSea
-- Install [rpm](https://build.opensuse.org/package/show/home:swiftgist/deepsea)
-- For non-RPM-distros, try `make install`.
-
-### Configure
-- DeepSea needs to know which node will act as the Salt master.
-  To avoid any possible user confusion, DeepSea enforces that all minion
-  hostnames (including the `admin node`'s hostname) should match the minion id.
-  Edit [/srv/pillar/ceph/master_minion.sls](srv/pillar/ceph/master_minion.sls),
-  and ensure the value of `master_minion` matches the minion id for the
-  `admin node`. (In our example from __Prepare Salt__ above, the value should be
-  `admin.ceph`.)
-
-### Steps
-- Run `salt-run state.orch ceph.stage.0` or `salt-run state.orch ceph.stage.prep`
-- Run `salt-run state.orch ceph.stage.1` or `salt-run state.orch ceph.stage.discovery`
-- Create `/srv/pillar/ceph/proposals/policy.cfg`.  Examples are [here](doc/examples)
-- Run `salt-run state.orch ceph.stage.2` or `salt-run state.orch ceph.stage.configure`
-- Run `salt-run state.orch ceph.stage.3` or `salt-run state.orch ceph.stage.deploy`
-- Run `salt-run state.orch ceph.stage.4` or `salt-run state.orch ceph.stage.services`
-
-### Details on policy.cfg
-The discovery stage (or stage 1) creates many configuration proposals under
-`/srv/pillar/ceph/proposals`. The files contain configuration options for Ceph
-clusters, potential storage layouts and role assignments for the cluster
-minions. The policy.cfg specifies which of these files and options are to be
-used for the deployment.
-
-Please refer to the [Policy wiki page](https://github.com/SUSE/DeepSea/wiki/policy)
-for more detailed information.
+You need at least a minimum of 4 nodes to be able to test and use DeepSea properly.
+You could set the option `DEV_ENV=true` during the deployment stages to be able to use
```

Comment:
I'd agree that it's not intuitive that works as either an environment variable or a pillar variable.